src/App.jsx

```diff
@@ -19,10 +19,7 @@ const HomeScreenLayout = ({ children }) => {
     <>
       <Header onToggleSidebar={ToggleSidebarHandler} />
       <div className="app__container">
-        <SideBar
-          isSidebarVisible={isSidebarVisible}
-          onToggleSidebar={ToggleSidebarHandler}
```

Comment:
function name. I approved with an assumption that you will rename it correctly and then merge. I see wrong update pf the name here @Tanvir04khan

Reply:
changed the function's name